.github/Branching.md

```diff
@@ -0,0 +1,145 @@
+  - Upon merging into the <span style="color: #FFB366;font-weight:bold">develop</span> (QA) branch, a QA release will be automatically performed.
+  - Upon merging into the <span style="color: #82B366;font-weight:bold">master</span> (Production) branch, a production release will be automatically performed.
+
+
+<h3 align="left" style="color: #CC99FF;font-weight:bold">Preview Branches</h3>
+
+Holds minimal changes for upcoming production release stability.
+
+**_NOTE_:** Used for refactoring, bug fixes, and changes related to making an upcoming release more stable and to give users the chance to utilize the software and provide feedback before a major release.  Introducing major features outside of the changes in the upcoming release is not allowed. These kinds of changes are performed on the <span style="color: #CC99FF;font-weight:bold">preview</span> branch by using <span style="color: #9E269E;font-weight:bold">preview feature</span> branches.
+- **Branch Syntax:** preview/v\<major\>.\<minor\>.\<patch\>-preview.\<prev number\>
+  - Example: preview/v1.2.3-preview.4
+- **Branches That Can Merge Into Preview Branches:** <span style="color: #9E269E;font-weight:bold">preview feature</span> branches
+- **Created From:** <span style="color: #00CCCC;font-weight:bold">release</span> branches
+- **Merged Into:** <span style="color: #00CCCC;font-weight:bold">release</span> branches
+- **Environment:** none
+- **Required Casing:** all lowercase
+- **CI/CD:**
+  - The major, minor, and patch numbers of the preview branch and the release branch it was created from, must match. 
+  - Build, unit test, and version validation status checks are automatically run for each change to the pull request.
+  - All status checks must pass before a pull request will be completed.
+
+
+<h3 align="left" style="color: #9E269E;font-weight:bold">Preview Feature Branches</h3>
+
+Where a developer's work will be performed when implementing changes for a <span style="color: #CC99FF;font-weight:bold">preview</span> branch via a pull request.
+- **Branch Syntax:** preview/feature/\<issue id\>-\<description\>
+  - Example: preview/feature/123-my-branch
+- **Branches That Can Merge Into Preview Feature Branches:** none
+- **Created From:** <span style="color: #CC99FF;font-weight:bold">preview</span> branches
+- **Merged Into:** <span style="color: #CC99FF;font-weight:bold">preview</span> branches
+- **Environment:** none
+- **Required Casing:** all lowercase
+- **CI/CD:**
+  - Build and unit test status checks are automatically run for each change to the pull request.
+  - All status checks must pass before a pull request will be completed.
+
+---
+
+<h2 align="center">
+   <div>
+      <span style="font-weight:bold">Branching Diagram</span>
+   </div>
+
+![BranchingDiagram](./Images/BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
+![BranchingDiagram](./Images/BranchingDiagram-LightMode-v1.1.png#gh-light-mode-only)
```

Comment:
Both of these diagrams need to be moved over to the root of this repo as well.

These links also need to be updated to work properly.
Use the files from Velaptor and copy them into the root.

Refer to below:

```suggestion
![BranchingDiagram](./BranchingDiagram-DarkMode-v1.1.png#gh-dark-mode-only)
![BranchingDiagram](./BranchingDiagram-LightMode-v1.1.png#gh-light-mode-only)
```